gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.6-all.zip
```

Comment:
Can we also add distributionSha256Sum?

This is a Gradle security best practice (prevents tampered downloads). Also ensure gradlew is executable in Git. Checksums are on the Gradle releases pages.